test/controllers/BulkRequestReceivedControllerSpec.scala

```diff
@@ -70,118 +71,84 @@ class BulkRequestReceivedControllerSpec extends PlaySpec with OneServerPerSuite
 
   "BulkRequestReceivedController" must {
 
-    "respond to GET /guaranteed-minimum-pension/request-received" in {
-      val result = route(FakeRequest(GET, "/guaranteed-minimum-pension/request-received"))
-      status(result.get) must not equal (NOT_FOUND)
-    }
-
     "request recevied GET " must {
 
-      "be authorised" in {
-        getBulkRequestReceived() { result =>
-          status(result) must equal(SEE_OTHER)
-          redirectLocation(result).get must include("/gg/sign-in")
-        }
-      }
-
       "authenticated users" must {
 
         "respond with ok" in {
 
           when(mockSessionService.fetchGmpBulkSession()(Matchers.any(), Matchers.any())).thenReturn(Future.successful(Some(gmpBulkSession)))
           when(mockBulkRequestCreationService.createBulkRequest(Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any())).thenReturn(Left(bulkRequest1))
-          when(mockGmpBulkConnector.sendBulkRequest(Matchers.any())(Matchers.any(),Matchers.any())).thenReturn(Future.successful(OK))
-          withAuthorisedUser { user =>
-            getBulkRequestReceived(user) { result =>
+          when(mockGmpBulkConnector.sendBulkRequest(Matchers.any(),Matchers.any())(Matchers.any())).thenReturn(Future.successful(OK))
+
+          val result = TestBulkRequestReceivedController.get.apply(FakeRequest())
```

Comment:
you should never need a literal apply TestBulkRequestReceivedController.get(FakeRequest()) is synonymous, expanded by the compiler

Reply:
Done now